client-ui/src/app/components/time-frame-picker/date-range-picker/date-range-picker.component.ts

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2019 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {Component, EventEmitter, Input, Output} from '@angular/core';
+import {NgbDateStruct} from "@ng-bootstrap/ng-bootstrap";
+import * as _ from "lodash"
+
+type Model = [NgbDateStruct, NgbDateStruct]
+type BsModel = [Date, Date]
+
+@Component({
+  selector: 'date-range-picker',
+  templateUrl: './date-range-picker.component.html'
+})
+export class DateRangePickerComponent {
+  public bsMinDate: Date
+  public bsMaxDate: Date
+  public bsModel: BsModel
+
+  @Input()
+  public set minDate(ngbDate: NgbDateStruct) {
+    this.bsMinDate = DateRangePickerComponent.structToDate(ngbDate)
+    console.log("@Input() min date", this.bsMinDate)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Calls to 'console.log' are not allowed.](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4491035)